0-bootstrap/main.tf

```
@@ -30,8 +30,7 @@ resource "google_folder" "bootstrap" {
 }
 
 module "seed_bootstrap" {
-  source                         = "terraform-google-modules/bootstrap/google"
-  version                        = "~> 5.0"
+  source                         = "github.com/terraform-google-modules/terraform-google-bootstrap/modules/cloudbuild"
```

```suggestion
  source                         = "github.com/terraform-google-modules/terraform-google-bootstrap"
```
@iyabchen  It needs to be the main bootstrap module for the seed module